### PR TITLE
Create pt.json

### PR DESCRIPTION
**Pull request by ViPeR5000**

### CODE REVIEW

File: custom_components/ryanair/translations/pt.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Equipamento já configurado"
```
Comment (jampez77) [inline custom_components/ryanair/translations/pt.json:4]: According to Google this  translate to "Equipment already configured". If that's the case, is "Dispositivo já configurado" a truer translation?

File: custom_components/ryanair/translations/pt.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        },
+        "error": {
+            "cannot_connect": "Falha na ligação",
+            "invalid_auth": "Autenticação invalida",
+            "invalid_mfa_code": "Codigo MFA errado",
```
Comment (jampez77) [inline custom_components/ryanair/translations/pt.json:9]: According to Google this translate to "Wrong MFA code". If that's the case, is "Código MFA inválido" a truer translation?

File: custom_components/ryanair/translations/pt.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                "data": {
+                    "mfa_code": "Código MFA (8-digitos)"
+                },
+                "description": "{email}",
+                "title": "Inserir o código MFA"
+            },
+            "user": {
+                "data": {
+                    "email": "Email",
+                    "host": "servidor",
```
Comment (jampez77) [inline custom_components/ryanair/translations/pt.json:23]: According to Google this translate to "Server". If that's the case, is "hospedar" a truer translation?

File: custom_components/ryanair/translations/pt.json (hunk 1 above)
Comment (ViPeR5000) [reply custom_components/ryanair/translations/pt.json:4]: Yes

File: custom_components/ryanair/translations/pt.json (hunk 3 above)
Comment (ViPeR5000) [reply custom_components/ryanair/translations/pt.json:23]: Hospedar = Portuguese from brasil.

File: custom_components/ryanair/translations/pt.json (hunk 2 above)
Comment (ViPeR5000) [reply custom_components/ryanair/translations/pt.json:9]: wrong == Errado or Inválido is the same meaning

Comment (jampez77) [reply custom_components/ryanair/translations/pt.json:9]: Fair enough. I'm happy with this then.

File: custom_components/ryanair/translations/pt.json (hunk 3 above)
Comment (jampez77) [reply custom_components/ryanair/translations/pt.json:23]: `servidor` is fine in that case.

File: custom_components/ryanair/translations/pt.json (hunk 1 above)
Comment (jampez77) [reply custom_components/ryanair/translations/pt.json:4]: In that case could you update this?

**Pull request merged**